web/src/sidebar_ui.js

```diff
@@ -28,6 +28,14 @@ export function hide_streamlist_sidebar() {
     $(".app-main .column-left").removeClass("expanded");
 }
 
+export function any_sidebar_expanded() {
```

Comment:
So I'm thinking a bit more about this name, and `expanded` may not quite mean what we want -- as one might think it'd be true for a really wide window. Maybe `any_sidebar_shown_as_overlay` or something?

Reply:
Ok, done